projects/core/src/anonymous-consents/facade/anonymous-consents.service.ts

```diff
@@ -128,9 +128,11 @@ export class AnonymousConsentsService {
    */
   giveAllConsents(): Observable<ConsentTemplate[]> {
     return this.getTemplates().pipe(
-      tap(templates =>
-        templates.forEach(template => this.giveConsent(template.id))
-      )
+      tap(templates => {
+        if (templates) {
```

Comment:
I guess this would essentialy be the same as a `filter(Boolean)`

Reply:
Yes, good point